deploy-server/config/nginx-site.conf

```diff
@@ -28,7 +28,13 @@
 
         ssl_certificate     ssl/stocks.cert.pem;
         ssl_certificate_key ssl/stocks.key.pem;
-        ssl_dhparam         dhparam.pem;
+
+        # For maximal security you might want to generate the DH
+        # parameters yourself using 'openssl'.
+        #   openssl dhparam -out /etc/nginx/dhparam.pem 4096
+        # This might take a while though.
+        #ssl_dhparam         dhparam.pem;
```

Comment:
This is unlikely to be found here. Please put it either into `manual/server/maintainance.md` [sic] or `manual/server/server-installation.md`. Please also add a link to the doc of the directive at nginx. Also reference `man 1 dhparam`